core/controllers/profile.py

```diff
@@ -311,6 +343,39 @@ def get(self):
 class SignupHandler(base.BaseHandler):
     """Provides data for the editor prerequisites page."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'username': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_valid_username_string'
+                    }]
+                }
+            },
+            'agreed_to_terms': {
+                'schema': {
+                    'type': 'bool'
+                },
+                'default_value': None
+            },
+            'default_dashboard': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
```

Comment:
Why default `None`?

Reply:
After reading the wiki page, I have updated this. Now is it ok?